src/basics.md

```diff
@@ -412,6 +414,80 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-glob-recursive]: #ex-glob-recursive
+<a name="ex-glob-recursive"></a>
+## Find all png files recursively
+
+[![glob-badge]][glob] [![cat-filesystem-badge]][cat-filesystem]
+
+Recursively find all png files using the [glob] crate.
```

Comment:
I would suggest dropping the `using glob crate part`. Reader should be already informed about it thanks to badges/code snippet and other links in the description. Honestly the surrounding examples should be also corrected to follow this rule.